.github/workflows/ci-testing.yml

```diff
@@ -1,14 +1,18 @@
 name: CI-Code-Style
 
-on: [push]
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
 
 jobs:
   unit_tests:
-    name: Linter checks
-    runs-on: ubuntu-20.04
+    name: Unit tests for cloud regionsrv client
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6]
+        python-version: ["3.11"]
```

Comment:
cloud-regionsrv-client still only builds for Python 3.6. I think we should make this change when/if we switch the package build to Python 3.11

Reply:
my understanding when we talked about the PR, was that a new test env would be aded
so one could run `tox -e unit_py3_6` or `tox -e unit_py3_11` not to remove 3.6 which we still need AFAIK

Reply:
nothing got removed. With this PR you have now two workflow files, one for legacy (3.6) and one for the future 3.11 onwards. I was not able to handle 3.6 + 3.11 + more in one workflow file because the ubuntu version that is used to run the containers for the workflow run does not offer all this python versions. In fact py3.6 only exists for an older ubuntu container which we select in the workflow definition to test the 3.6 run and for 3.11 we have to select a newer ubuntu container but that one does not longer provide 3.6. I think it is not possible to select a different container per python version in one workflow file

Reply:
got it, it is in legacy